server/rest/src/main/java/org/infinispan/rest/JSONConstants.java

```diff
@@ -14,4 +14,7 @@
    String QUERY_STRING = "query";
    String TOTAL_RESULTS = "total_results";
    String TYPE = "_type";
+   String TEMPLATE = "template";
```

Comment:
Technically those are not JSONConstants, so it'd be better to either rename this interface or move those 3 new values to another place